grai-integrations/source-snowflake/src/grai_source_snowflake/models.py

```diff
@@ -0,0 +1,152 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+from pydantic import BaseModel, Field, root_validator, validator
+
+
+class SnowflakeNode(BaseModel):
+    pass
+
+
+class ID(SnowflakeNode):
+    name: str
+    namespace: str
+    full_name: str
+
+
+class TableID(ID):
+    table_schema: str
+
+    @root_validator(pre=True)
+    def make_full_name(cls, values):
```

Comment:
I'm not a Snowflake expert but the connection parameters seemed to depend on an account, warehouse, and role parameter on top of the schema. Would any of those matter in creating unique IDs on tables/columns?